models/README.md

```diff
@@ -2,6 +2,11 @@
 
 This directory contains serializations of models in various formats, m.n. the LIonWeb JSON and Ecore XML formats:
 
+* [`library.ecore`](./library.ecore): Ecore XML file with the “library” Ecore metamodel - copied from [here in the `lioncore-java` repository](https://github.com/LIonWeb-org/lioncore-java/blob/master/emf/src/test/resources/library.ecore).
+* [`library.ecore.json`](./library.ecore.json): a serialization in JSON format of the result of parsing the `library.ecore` using Deno's XML functionality.
```

Comment:
Is it useful to commit this file?

Reply:
Which one do you mean?:

- `library.ecore` is needed to test the importer.
- `library.ecore.json` is Git-ignored, so not committed.